src/main/java/org/beta/authentification/matrix/rest/MatrixServiceFactory.java

```diff
@@ -34,23 +34,21 @@ private static MatrixService getNonSingletonInstance() {
         return new MatrixService(homeServerList, unauthorizedList);
     }
 
-    public static MatrixService getAuthenticatedInstance() {
+    public static MatrixService getAuthenticatedInstance(String accountEmail, String password) {
```

Comment:
I would rather add a other method with additional parameters than modify the existing code